packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -315,7 +371,7 @@ class Auth {
         }
         const accessToken = this.get('access_token')
 
-        if (accessToken && !this.isTokenExpired(accessToken)) {
+        if (accessToken && this.isTokenValid(accessToken)) {
```

Comment:
```suggestion
        if (accessToken && this.isTokenValid(accessToken, isGuest)) {
```

Reply:
Updated code to use isGuest to determine cookie to fetch.